### PR TITLE
Skat direction feature

### PR DESCRIPTION
**Pull request by aateg**

### CODE REVIEW

**Review by Yamaguti — APPROVED** [review]
Overall this is looking good.
Maybe you need to setup automatic conversion from tab to spaces in VSCode?

github file visualitation uses 4 spaces for a tab , and IIUC you used 2 on your personal VSCode settings, which is causing some seemingly wrong indentation on github.

I think SBKing needs to set a standard, whether we all use tabs or spaces

**Comment by aateg** [comment]
Hi @Yamaguti, thanks for the review! I don't know yet how to fix that identation problem, if you guys know how to set that configuration in vscode, please let me know. But I'm going to wait for @rulojuka comment about this project pattern, then fix it before merge.

File: src/test/java/br/com/sbk/sbking/core/ThreePlayersDirectionTest.java
Comment (rulojuka) [reply src/test/java/br/com/sbk/sbking/core/ThreePlayersDirectionTest.java:25]: Not yet, there is an open issue for that #13 . I kind of think that enforcing this kind of style without something automated to be really bad. For now, the "rule" is that everything that you did not really change should not appear on the git diff :)

**Review by rulojuka — APPROVED** [review]
The english names kind of seem strange to me, as I guess german names will be known to all skat players but english may not. Anyway, that is your call.
About the spaces and tabs, forget about that for now. I will enforce it soon with some kind of linter. Just make sure that no lines without real changes appear on the git diff.

If you want a temporary solution, the one I use is adding `"editor.formatOnSave": true` on my settings.json on vscode and the `Java Extension Pack` will install the same linter as mine (which I guess is `Language Support for Java(TM) by Red Hat` but comes with the Extension Pack.

**Pull request merged**
